ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/MultiPartyServiceCallAuthTests.scala

```diff
@@ -104,7 +104,9 @@ trait MultiPartyServiceCallAuthTests extends SecuredServiceCallAuthTests {
 
   it should "deny single-party calls authorized to no parties (party)" taggedAs securityAsset
     .setAttack(
-      attack(threat = "Exploit a single-party call authorized to no parties (party)")
+      attackPermissionDenied(threat =
+        "Exploit a single-party call authorized to no parties (party)"
```

Comment:
```suggestion
        "Execute a single-party call authorized to no parties (party)"
```

as `exploit` is very generic again.

If you decide to accept this suggestion please also change the other occurrences below.

Reply:
ah missed, this file, will fix